app/code/Magento/InventoryInStorePickup/etc/module.xml

```diff
@@ -6,5 +6,10 @@
  */
 -->
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
-    <module name="Magento_InventoryInStorePickup" setup_version="1.0.0" />
+    <module name="Magento_InventoryInStorePickup" setup_version="1.0.0">
+        <sequence>
+            <module name="Magento_Sales" />
+            <module name="Magento_Inventory" />
```

Comment:
Should it be `Magento_Inventory` or `InventoryApi`?

Also `sales` should be added to composer dependencies

Reply:
For now, dependency on Magento_Sales only.
Added to composer.json.